app/forms/idv/api_image_upload_form.rb

```diff
@@ -260,47 +216,17 @@ def extra_attributes
         flow_path: params[:flow_path],
       }
 
-      @extra_attributes[:front_image_fingerprint] = front_image_fingerprint
-      @extra_attributes[:back_image_fingerprint] = back_image_fingerprint
-      @extra_attributes[:passport_image_fingerprint] = passport_image_fingerprint
-      @extra_attributes[:selfie_image_fingerprint] = selfie_image_fingerprint
+      images.each do |image|
+        @extra_attributes[image.extra_attribute_key] = image.fingerprint
+      end
+
       @extra_attributes[:liveness_checking_required] = liveness_checking_required
       @extra_attributes[:document_type] = document_type
       @extra_attributes
     end
 
-    def front_image_fingerprint
-      return @front_image_fingerprint if @front_image_fingerprint
-      if readable?(:front)
-        @front_image_fingerprint =
-          Digest::SHA256.urlsafe_base64digest(front_image_bytes)
-      end
-    end
-
-    def back_image_fingerprint
-      return @back_image_fingerprint if @back_image_fingerprint
-      if readable?(:back)
-        @back_image_fingerprint =
-          Digest::SHA256.urlsafe_base64digest(back_image_bytes)
-      end
-    end
-
-    def passport_image_fingerprint
-      return @passport_image_fingerprint if @passport_image_fingerprint
-      if readable?(:passport)
-        @passport_image_fingerprint =
-          Digest::SHA256.urlsafe_base64digest(passport_image_bytes)
-      end
-    end
-
     def selfie_image_fingerprint
-      return unless liveness_checking_required
```

Comment:
I suspect this deletion is the reason for https://github.com/18F/identity-idp/pull/12197/files#r2100744980